docker/drama-free-django/test.sh

```diff
@@ -1,3 +1,11 @@
 #!/usr/bin/env bash
 
-docker run -v `pwd`:/cfgov centos:6 /cfgov/docker/drama-free-django/_test.sh
+set -e
+
+docker build -t cfgov-dfd-builder docker/drama-free-django
+
+docker run \
+  --rm \
+  -u $(id -u):$(id -g) \
+  -v $(pwd):/cfgov:cached \
+  cfgov-dfd-builder ./_test.sh
```

Comment:
This shouldn't hold up this PR, but, if we are going to maintain this DFD testing capability, I wonder if it's worth entertaining the idea of a distinct Dockerfile just for that purpose. We don't really need everything in the "cfgov-dfd-builder" image just to run the DFD image, and it would be nice to actually determine what is needed. But probably thinking more about that should wait until/if we want to think about migrating Ansible code here.

Reply:
Yeah, it does feel a little awkward to have the two scripts that both build the same image, but my assumption was that _generally_ when you'd run `test.sh`, you would have run `build.sh` just before it, and so the `docker build` part would be fully cached and be pretty instant, but I didn't want to make that a requirement to running `test.sh`, so it's duplicated  in both scripts.

As for a separate image, yeah we could.  It just seemed like it'd be better to only maintain one Dockerfile that could do both.  But if there's a scenario where we'd be running _just_ `test.sh`, maybe that starts to make more sense.